onnx/test/test_external_data.py

```diff
@@ -368,6 +368,51 @@ def test_save_model_without_loading_external_data(self) -> None:
         self.assertFalse(attribute_tensor.HasField("data_location"))
         self.assertTrue(np.allclose(to_array(attribute_tensor), self.attribute_value))
 
+    def test_save_model_as_external_data_automatically(self) -> None:
```

Comment:
Please consider using onnx model script https://github.com/onnx/onnx/blob/main/onnx/test/parser_test.py for better readability.

Reply:
Thanks! Will update soon